testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -13519,6 +13519,46 @@ void testTimestampDiff(boolean coercionEnabled) {
     f.checkNull("date_add(CAST(NULL AS DATE), interval 5 day)");
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6396">[CALCITE-6396]
+   * Add ADD_MONTHS function enabled in Oracle, Spark library)</a>.
+   */
+  @Test void testAddMonths() {
+    final SqlOperatorFixture f0 = fixture()
+        .setFor(SqlLibraryOperators.ADD_MONTHS);
+    f0.checkFails("^add_months(date '2008-12-25', "
+            + "5)^",
+        "No match found for function signature "
+            + "ADD_MONTHS\\(<DATE>, <NUMERIC>\\)", false);
+    final Consumer<SqlOperatorFixture> consumer = f -> {
+      f.checkScalar("add_months(date '2016-02-22', 2)",
```

Comment:
Please add tests like `select ADD_MONTHS('2016-08-31',1);`  
In Spark, this SQL will return
```
spark-sql (default)> select  ADD_MONTHS('2016-08-31',1);
2016-09-30
Time taken: 0.118 seconds, Fetched 1 row(s)
```

Reply:
done